set*uid() is not implemented but it does not matter anyway because there
is only one user per noux instance.

+++ src/noux-pkg/openssh/sshconnect.h
@@ -55,6 +55,7 @@ void	 ssh_userauth2(const char *, const char *, char *, Sensitive *);
 void	 ssh_put_password(char *);
 int	 ssh_local_cmd(const char *);
 
+#if 0
 /*
  * Macros to raise/lower permissions.
  */
@@ -73,3 +74,6 @@ int	 ssh_local_cmd(const char *);
 		    strerror(errno));			\
 	errno = save_errno;				\
 } while (0)
+#endif
+#define PRIV_START
+#define PRIV_END
